src/util/bin/cosi-graph-update.js

```diff
@@ -17,6 +17,7 @@ const Graph = require(path.join(cosi.lib_dir, 'graph'));
 app.
     version(cosi.app_version).
     usage('[options] <config_file>').
+    description('Update Circonus graph from local graph config file (/opt/circonus/cosi/registration/registration-*.json)').
```

Comment:
i wouldn't use the registration directory in the example. just something like `graph_configuration.json` remember, the registration directory is _documentation_ on what happened during the registration process. we shouldn't encourage doing things _in_ that directory. making a copy of a file, editing it, and using an update command is ok though.